index.android.js

```diff
@@ -525,6 +525,26 @@ class RNGoogleFit {
     return result;
   }
 
+  getBloodGlucoseSamples = async (options, callback) => {
+    const { startDate, endDate, bucketInterval, bucketUnit } = prepareInput(options);
+    const result = await googleFit.getBloodGlucoseSamples(
+      startDate,
+      endDate,
+      bucketInterval,
+      bucketUnit,
+    );
+    if (result.length > 0) {
+      return prepareResponse(result).map((i) => ({...i, value: i.value.toFixed(3)}));
```

Comment:
since user will use the data to display, maybe leave the `toFixed(3)`  to the, let them figure out whatever they want, 
`toFixed()` also change the datatype from `number` to `string`

Reply:
not understand the part
```
 leave the toFixed(3) to the, let them figure out whatever they want,
 ```
 
 what did you mean? Need to remove `toFixed(3)` ?

Reply:
Irrelevant to this original question, just a slight interesting thing to share,  `prepareResponse().map()` is `O(2n)` (or O(n) technically) operation due to double iteration.
Theoretically it should have no effect, and today's device is powerful. But usually user will do their own iteration based on the data so generally we expect there will be multiple iterations happens for same dataset.  
When the amount of data is slightly large (for example, I have a 1.5 years step data while testing the app) and we want to use it to render on the screen. This side optimization may worth it, but maybe I'm worried too much.

Reply:
And if user want to further modify the dataset, they can do whatever they want within one iteration along with this format.

Reply:
@aboveyunhai make sense, already removed, but anyway I'm sure you will prettify those values from google fit after getting it

Reply:
> not understand the part
> 
> ```
>  leave the toFixed(3) to the, let them figure out whatever they want,
> ```
> 
> what did you mean? Need to remove `toFixed(3)` ?

just direct `return return prepareResponse(result);` unless I mistaken something